lib/Doctrine/DBAL/Platforms/AbstractPlatform.php

```diff
@@ -853,9 +853,9 @@ public function getLowerExpression($str)
     /**
      * Returns the SQL snippet to get the position of the first occurrence of substring $substr in string $str.
      *
-     * @param string          $str      Literal string.
-     * @param string          $substr   Literal string to find.
-     * @param integer|boolean $startPos Position to start at, beginning of string by default.
+     * @param string      $str      Literal string.
+     * @param string      $substr   Literal string to find.
+     * @param int|bool $startPos Position to start at, beginning of string by default.
```

Comment:
Please fix alignment.

Reply:
I'm getting an error locally:
```bash

FILE: lib/Doctrine/DBAL/Platforms/AbstractPlatform.php
----------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------
 858 | ERROR | [x] Expected 1 spaces after parameter type; 4 found
----------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------
```
Update nevertheless?

Reply:
You should un-indent the $str and $substr params.